tools/roxvet/common/file_filters.go

```diff
@@ -15,14 +15,14 @@ func IsGeneratedFile(f *ast.File) bool {
 	if len(f.Comments) == 0 {
 		return false
 	}
-	firstComment := f.Comments[0].List[0]
-
-	if firstComment.Pos() > f.Package {
-		// comment is not at beginning of file, before the package keyword
-		return false
+	for _, comments := range f.Comments {
+		for _, comment := range comments.List {
+			if generatedCodeRegex.MatchString(comment.Text) {
+				return true
```

Comment:
Should we still keep checking that it is before the package?
```suggestion
				return comment.Pos() < f.Package
```


Another option is to check it before `if` and return if the comment position is after `f.Package`. That would also break the loop earlier. But we can do that only if we are sure that the order of comments is correct.